package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.4.1",
+    "chai": "^4.3.4",
+    "hardhat": "^2.8.0"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.3"
+  }
```

Comment:
Recommend adding a `scripts` section with pre-configured commands for `test`, `compile`, and `deploy`, so that devs don't have to futz with `npx hardhat` commands.

Reply:
https://github.com/steem-monsters/sps-minter/commit/644429cec057b00ea4a3d7f212e8cf2b6b931129